guides/source/action_controller_overview.md

```diff
@@ -374,11 +374,14 @@ own code to adapt to your situation.
 Session
 -------
 
-Your application has a session for each user in which you can store small amounts of data that will be persisted between requests. The session is only available in the controller and the view and can use one of several of different storage mechanisms:
+Your application has a session for each user in which you can store small
+amounts of data that will be persisted between requests. The session is only
+available in the controller and the view. The storage mechanism can be
+configured to a store provided by an external gem or one that Rails provides by
+default:
 
 * [`ActionDispatch::Session::CookieStore`][] - Stores everything on the client.
 * [`ActionDispatch::Session::CacheStore`][] - Stores the data in the Rails cache.
-* `ActionDispatch::Session::ActiveRecordStore` - Stores the data in a database using Active Record (requires the `activerecord-session_store` gem).
```

Comment:
The `activerecord-session_store` gem is still being maintained under the `rails` org, so let's keep this.  We can link to https://github.com/rails/activerecord-session_store though, and lean on its README for documentation purposes.

Reply:
Hmm, I tried to find some context on the reasoning for extraction to a gem but came up short. I was curious what prompted it and whether that meant it should be recommended/documented.